src/api/eventTypes.js

```diff
@@ -173,3 +175,253 @@ export type RestartEvent = $ReadOnly<{|
+  inline_image_preview:
+    $PropertyType<InitialDataRealm, 'realm_inline_image_preview'>,
+  inline_url_embed_preview:
+    $PropertyType<InitialDataRealm, 'realm_inline_url_embed_preview'>,
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_invite_by_admins_only'>,
+  invite_required:
+    $PropertyType<InitialDataRealm, 'realm_invite_required'>,
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_invite_to_realm_policy'>,
+  invite_to_stream_policy:
+    $PropertyType<InitialDataRealm, 'realm_invite_to_stream_policy'>,
+  is_zephyr_mirror_realm:
+    $PropertyType<InitialDataRealm, 'realm_is_zephyr_mirror_realm'>,
+  logo_source:
+    $PropertyType<InitialDataRealm, 'realm_logo_source'>,
+  logo_url:
+    $PropertyType<InitialDataRealm, 'realm_logo_url'>,
+  mandatory_topics:
+    $PropertyType<InitialDataRealm, 'realm_mandatory_topics'>,
+  message_content_allowed_in_email_notifications:
+    $PropertyType<InitialDataRealm, 'realm_message_content_allowed_in_email_notifications'>,
+  message_content_delete_limit_seconds:
+    $PropertyType<InitialDataRealm, 'realm_message_content_delete_limit_seconds'>,
+  message_content_edit_limit_seconds:
+    $PropertyType<InitialDataRealm, 'realm_message_content_edit_limit_seconds'>,
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_message_retention_days'>,
+  move_messages_between_streams_policy:
+    $PropertyType<InitialDataRealm, 'realm_move_messages_between_streams_policy'>,
+  realm_name:
+    $PropertyType<InitialDataRealm, 'realm_name'>,
+  name_changes_disabled:
+    $PropertyType<InitialDataRealm, 'realm_name_changes_disabled'>,
+  night_logo_source:
+    $PropertyType<InitialDataRealm, 'realm_night_logo_source'>,
+  night_logo_url:
+    $PropertyType<InitialDataRealm, 'realm_night_logo_url'>,
+  notifications_stream_id:
+    $PropertyType<InitialDataRealm, 'realm_notifications_stream_id'>,
+  password_auth_enabled:
+    $PropertyType<InitialDataRealm, 'realm_password_auth_enabled'>,
+  plan_type:
+    $PropertyType<InitialDataRealm, 'realm_plan_type'>,
+  presence_disabled:
+    $PropertyType<InitialDataRealm, 'realm_presence_disabled'>,
+  private_message_policy:
+    $PropertyType<InitialDataRealm, 'realm_private_message_policy'>,
+  push_notifications_enabled:
+    $PropertyType<InitialDataRealm, 'realm_push_notifications_enabled'>,
+
+  // TODO: Do servers send this, here or in InitialDataRealm?
+  //   realm_restricted_to_domain
+
+  send_welcome_emails:
+    $PropertyType<InitialDataRealm, 'realm_send_welcome_emails'>,
+
+  // TODO: Do servers send this, here or in InitialDataRealm?
+  //   realm_show_digest_email
+
+  signup_notifications_stream_id:
+    $PropertyType<InitialDataRealm, 'realm_signup_notifications_stream_id'>,
+  upload_quota:
+    $PropertyType<InitialDataRealm, 'realm_upload_quota'>,
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_upload_quota_mib'>,
```

Comment:
Another case where the API docs on the event side have the old name, and I suspect are wrong.